packages/calypso-apps-builder/index.js

```diff
@@ -74,10 +74,13 @@ async function runBuilder( args ) {
 	await Promise.all( [
 		runAll( [ `build:*${ watch ? ' --watch' : '' }` ], runOpts ).then( () => {
 			console.log( 'Build completed!' );
-			if ( ! watch && sync ) {
-				// In non-watch + sync mode, we sync only once after the build has finished.
-				setupRemoteSync( localPath, remotePath );
-			}
+			const translate = runAll( 'translate', runOpts ).catch( () => {} );
```

Comment:
It may be useful to at least log any errors.

Reply:
The errors will be logged because `runOpts` has `stderr` as the output. The only possible failure here is that the `translate` command doesn't exist.

Reply:
Yup, it logs errors inside the translate script. 

```
[translate] command not found: asdsadasdasdasd
```